### PR TITLE
DX-2020: Add Click to Pay

### PR DESCRIPTION
**Pull request by arebra**
Added mentions of click to pay in text and paid response in  v3 resource and status models.

### CODE REVIEW

File: _includes/checkout-v3-status-models.md [hunk 1]
```diff
@@ -513,6 +513,43 @@ Content-Type: application/json
...
+         "acquirerTransactionType": "WALLET",
+         "acquirerStan": "392",
+         "acquirerTerminalId": "80100001190",
+         "acquirerTransactionTime": "2022-09-05T06:45:40.322Z",
+         "transactionInitiator": "CARDHOLDER"
+         "bin": "489537"
+     }
+   }
+ }
+ ```
```
Comment (asbjornu) [inline _includes/checkout-v3-status-models.md:551]:
````suggestion
```http
HTTP/1.1 200 OK
Content-Type: application/json
{
  "paymentOrder": "/psp/paymentorders/5adc265f-f87f-4313-577e-08d3dca1a26c",
  "paid": {
    "id": "/psp/paymentorders/1f8d409e-8d8c-4ba1-a3ab-08da8caf7918/paid",
    "instrument": "ClickToPay",
    "number": 80100001190,
    "payeeReference": "1662360210",
    "amount": 1500,
    "details": {
      "nonPaymentToken": "12345678-1234-1234-1234-1234567890AB",
      "externalNonPaymentToken": "1234567890",
      "cardBrand": "Visa",
      "cardType": "Credit",
      "expiryDate": "12/0023",
      "issuerAuthorizationApprovalCode": "L00392",
      "acquirerTransactionType": "WALLET",
      "acquirerStan": "392",
      "acquirerTerminalId": "80100001190",
      "acquirerTransactionTime": "2022-09-05T06:45:40.322Z",
      "transactionInitiator": "CARDHOLDER"
      "bin": "489537"
    }
  }
}
```
````

**Review by asbjornu — CHANGES_REQUESTED** [review]
Some wrinkles with the indentation, otherwise 👍🏼

File: _includes/checkout-v3-resource-models.md [hunk 2]
```diff
@@ -654,6 +654,43 @@ Content-Type: application/json
...
+ Please note that this is an abbreviated example. See the main `paid` example for
+ more context.
+
+ {:.code-view-header}
+ **Click to Pay Response**
+
+ ```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
```
Comment (asbjornu) [inline _includes/checkout-v3-resource-models.md:668]:
```suggestion
Content-Type: application/json

{
```

File: _includes/checkout-v3-status-models.md [hunk 3]
```diff
@@ -513,6 +513,43 @@ Content-Type: application/json
...
+ Please note that this is an abbreviated example. See the main `paid` example for
+ more context.
+
+ {:.code-view-header}
+ **Click to Pay Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
```
Comment (asbjornu) [inline _includes/checkout-v3-status-models.md:527]:
```suggestion
Content-Type: application/json

{
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Sorry, I forgot this required newline in the previous review.

**Review by asbjornu — APPROVED** [review]

File: checkout-v3/index.md [hunk 4]
```diff
@@ -217,6 +218,7 @@ implementation.
   %}
 
 [apple-pay-logo]:/assets/img/applepay-logo.svg
+[c2p-logo]: /assets/img/clicktopay-logo.svg
```
Comment (asbjornu) [inline checkout-v3/index.md:221]: Does `/assets/img/clicktopay-logo.svg` already exist or did you forget to add it in this PR?

**Comment by arebra** [comment]
Added in Google Pay branch

**Pull request closed without merging**